internal/installation/stop.go

```diff
@@ -15,15 +22,33 @@ func StopRunning(installPath string) (rerr error) {
 		return errs.Wrap(err, "Could not get config")
 	}
 	defer rtutils.Closer(cfg.Close, &rerr)
-	
-	svcInfo := appinfo.SvcApp(installPath)
+
+	err = stopTray(installPath, cfg)
+	if err != nil {
+		return errs.Wrap(err, "Could not stop tray")
+	}
+
+	err = stopSvc(installPath)
+	if err != nil {
+		return errs.Wrap(err, "Could not stop service")
```

Comment:
Please make a localized error that explains to the user that they should manually kill the state-svc process and try again.